docker-compose.yml

```diff
@@ -166,14 +165,16 @@ services:
     labels:
       - "logging=true"
   agenda-approve:
-    image: kanselarij/agenda-approve-service:5.11.2
+    image: kanselarij/agenda-approve-service:feature-wait-on-busy-timeouts
```

Comment:
we probably don't want this. this disables concurrency check on agenda approving which would result in wrong VR numbers